lib/query_diet/widget.rb

```diff
@@ -42,8 +52,12 @@ def html(options)
     end
 
     module Helper
-      def query_diet_widget(options = {})
-        html = Widget.css + Widget.html(options)
+      def query_diet_widget(options = { 'nonce' => false })
+        with_nonce = options.fetch('nonce', options[:nonce])
+        nonce = with_nonce ? content_security_policy_nonce : nil
+        nonce_attribute = nonce ? " nonce=\"#{nonce}\"" : ''
```

Comment:
I would suggest to use a similar variant like in https://github.com/makandra/query_diet/blob/e9085f3709d9f75cd353534cf5394490da724fa5/lib/query_diet/logger.rb#L34. Otherwise `with_nonce` would be `nil` in case someone passes in an option hash like `{ :bad_count => 20 }`. 

And another note, which becomes irrelevant after the suggested change below: Using `nonce` as string and as symbol is not the same, both are different keys `{:nonce => false, 'nonce' => true}`. Only if you use a `hash_with_indifferent_access` (from Active Support) both are treated equal. 

Example:

```
def query_diet_widget(options = { })
  default_html_options = {:nonce => false}
  options = options.reverse_merge(default_html_options)

  nonce_attribute = options.fetch(:nonce) ? 'nonce="#{content_security_policy_nonce}"' : ''
```